docs/content/design/coordinator.md

```diff
@@ -188,6 +188,15 @@ Returns all rules for a specified datasource and includes default datasource.
 
  Returns audit history of rules. default value of interval can be specified by setting `druid.audit.manager.auditHistoryMillis` (1 week if not configured) in coordinator runtime.properties
 
+#### Namespaced Lookups
+
+* `/druid/coordinator/v1/lookups`
```

Comment:
For the seek of curiosity, it seems that in all the code the used name is namespace even for the module, why we would call this lookup.